manim/renderer/shaders/design_3.frag

```diff
@@ -0,0 +1,59 @@
+#version 330
+
+uniform vec3 u_resolution;
+uniform float u_time;
+out vec4 frag_color;
+
+vec3 palette(float d){
+	return mix(vec3(0.2,0.7,0.9),vec3(1.,0.,1.),d);
+}
+
+vec2 rotate(vec2 p,float a){
+	float c = cos(a);
+    float s = sin(a);
+    return p*mat2(c,s,-s,c);
+}
+
+float map(vec3 p){
+    for( int i = 0; i<8; ++i){
+        float t = u_time*0.2;
```

Comment:
The rotation was fast so maybe dial it down to `u_time*0.1` ?

Reply:
I made it slower.